### PR TITLE
Added guidance on adding images to docs

### PR DESCRIPTION
**Pull request by KyeMaloy97**
## Checklist 

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Adds some information to the docs to inform on how to add images to any doc updates and where to put the image files.

## Related Issues
#301

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -70,6 +71,14 @@ To add the doc to the side menu you must add it to the `sidebar.yaml` in `conten
...
 
+Images can help explain certain things better than words ever could, and make for more exciting and digestable content. To add an inline image to a doc, first place the image inside `content/docs/images` and use the following syntax:
+
+```
+![Helpful image](../images/my-awesome-image.png)
+```
+
+**Note:** relative paths must be used to reference image locations. If you can't get an image to show, you may be pointing to the wrong directory.
+
 2. Run the developement server
```
Comment (helenmasters) [inline DEVELOPMENT.md]: Whilst you're in here developement -> development

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -41,7 +41,8 @@ gatsby clean
 **Note:** If you are developing remotely, use `http://<hostname>:PORT` instead of `http://localhost:PORT` as described in this doc.
 
 ## Contributing Documentation
-1. Add documentation to the `content/docs` directory. This is where the documentation for [Appsody](https://appsody.dev/docs) is stored.
+1. Add documentation to the `content/docs` directory. This is where the documentation for [Appsody](https://appsody.dev/docs) is stored. 
+2. Add any  images to the `content/docs/images` directory.
```
Comment (helenmasters) [inline DEVELOPMENT.md]: V minor comment - there are two spaces in front of the word images.

Images can't have 2. here when there is another section further down called Running the development server which is also marked as 2...

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -70,6 +71,14 @@ To add the doc to the side menu you must add it to the `sidebar.yaml` in `conten
 ```
 **Note:** `title` for the section is optional but the `title` for each menu item is required.
 
+Images can help explain certain things better than words ever could, and make for more exciting and digestable content. To add an inline image to a doc, first place the image inside `content/docs/images` and use the following syntax:
```
Comment (helenmasters) [inline DEVELOPMENT.md]: Some slight rewording....

Images can help explain concepts better than words, and make for more exciting and digestible content. To add an inline image to a doc, first place the image inside `content/docs/images` and use the following syntax:
```
![Helpful image](../images/my-awesome-image.png)
```
**Note:** relative paths must be used to reference image locations. If an image does not render properly, you might be pointing to the wrong directory.

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -70,6 +71,14 @@ To add the doc to the side menu you must add it to the `sidebar.yaml` in `conten
...
+
+```
+![Helpful image](../images/my-awesome-image.png)
```
Comment (SueChaplain) [inline DEVELOPMENT.md]: Can we add more words to explain that "Helpful image" should contain alternative text?? E.g. [Add alternative text here to describe the image for users who are using a screen reader]

**Pull request merged**